package.json

```diff
@@ -34,5 +35,7 @@
     "parcel-bundler": "^1.12.4",
     "parcel-plugin-asset-copier": "^1.1.0"
   },
-  "devDependencies": {}
+  "devDependencies": {
+    "parcel-plugin-elm-i18n": "PedroHLC/parcel-plugin-elm-i18n"
```

Comment:
Does it make sense to call it like this if we won't use it as a parcel plugin?

Reply:
do you think it's beneficial having it as an external dependency instead of having as a script inside this repo?